source/introduction.txt

```diff
@@ -49,30 +49,28 @@ This frees you to focus on the features that makes your app special, not boilerp
 Serverless:
   - Deploy server-side applications without having to set up and manage server infrastructure. 
     {+service-short+} includes provisioning, deployment, operating systems, web servers, logging, backups, and redundancy.
-  - Run cloud-based :term:`{+service-short+} Functions <{+service-short+}
-    Function>` that can crunch data or interact with Node.js modules.
+  - Run cloud-based {+service-short+} Functions that can crunch data or interact with Node.js modules.
   - Skip writing an authentication API on top of your {+service+} service.
     Authenticated clients can interact with parts of the database based on their permissions.
 
 Dynamic and responsive:
-  - React to data changes in :term:`{+atlas+}`, process data from HTTPS endpoints,
+  - React to data changes in {+atlas+}, process data from HTTPS endpoints,
     or run {+service-short+} Functions on a schedule with 
-    :term:`{+service-short+} Triggers <{+service-short+} Trigger>`.
+    {+service-short+} Triggers .
   - Get up and running quickly for free, then scale according to the demands of your application.
   - Pay for and receive only the exact amount of compute you need at any given time
     with usage-based pricing. Usage under a certain amount per month is always free.
 
 Simply secure:
-  - Let users log in with familiar :term:`authentication providers
-    <authentication provider>` that are built into {+service+}.
+  - Let users log in with familiar authentication providers that are built into {+service+}.
   - Control which users may read and write data on a per-field basis with
     :ref:`role-based permissions <roles>` that you can define declaratively or
     with a function.
   - Enforce data integrity by defining :ref:`validation logic
-    <schemas>` with :term:`{+backend-schema+}`.
+    <schemas>` with {+backend-schema+}.
 
 Synchronized:
-  - Seamlessly sync data across devices with :term:`{+sync+} <{+sync+}>`.
+  - Seamlessly sync data across devices with {+sync+} .
   - Avoid writing complex synchronization logic with the :ref:`Database SDKs <realm-sdks>`.
 
 .. _intro-web-users:
```

Comment:
Can't leave a comment at the line, but in line 89 below, we say "{+backend-short+} SDK" which becomes "App Services SDK." I don't think that's in the scope of the renaming, is it? I didn't think we were renaming the Realm SDKs to "App Services SDKs". If not, we might want to look for anywhere we're using a source constant and make sure we don't follow it with "SDK" - otherwise, I don't think we can use the source constant.